pricing-page-exercise/salih/styles.css

```diff
@@ -0,0 +1,195 @@
+/* Global */
+
+*, ::after, ::before {
+    box-sizing: border-box;
+    font-family:sans-serif;
+}
+ul{
+    list-style-type: none;
+    padding: 0;
+    text-align: center;
+}
+.main li{
+    padding: 0.2rem 0;
+}
+h1{
+    margin: 0;
+    text-align: center;
+    margin-top: 5px;
```

Comment:
I'd recommend combining margin into one property: e.g. `margin: 5px 0 0` or `margin: 5px 0 0 0` (they both mean the same thing).

Reply:
Done